Assets/Scripts/EOSSessionsManager.cs

```diff
@@ -574,13 +632,31 @@ public void OnLoggedOut()
             userLoggedIn = false;
         }
 
+        /// <summary>
+        /// Unsubscribes from invitations and Peer2Peer message requests.
+        /// This should be called before the application finishes shutting down in order to properly remove Native notifications.
+        /// TODO: This function has no references, and is incomplete in its implementation. This will likely be removed.
```

Comment:
I think more properly it _should_ be called, as (just going based off of the contents of the function) it seems to be freeing some resources. I have noticed that this seems to be the case with several of the managers.

Reply:
This already uses the word "should". Would this be better as "must"?